README.md

```diff
@@ -111,7 +112,7 @@ The complete feature set includes the following.
 | Toggle the Beta badge         | `/beta`                           |
 | Get coordinates               | `/xyz`                            |
 | Toggle God mode               | `/god`                            |
-| God mode with team-killing    | `/demigod`                        |
+| Immune to melee and guns      | `/demigod`                        |
```

Comment:
This is not exactly true by the way. It includes every cause of death except for being left behind if I am not wrong.

Reply:
demigod can still die to mines, vomit, bracken, giant, i.e. any kind of death not depending from `DamagePlayer`

Reply:
You're right. I wonder what's a better description for this :/